pulumitest/replaceProgram.go

```diff
@@ -0,0 +1,20 @@
+package pulumitest
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func (a *PulumiTest) ReplaceProgram(program string) {
```

Comment:
The name is misleading. Only works for YAML? Not going to replace TypeScript. Perhaps "ReplacePulumiYAML"?

Reply:
Yeah excellent point actually, I've been using this exclusively for YAML, so missed this.